rest/data/commit_queue.go

```diff
@@ -88,6 +89,44 @@ func (pc *DBCommitQueueConnector) CommitQueueClearAll() (int, error) {
 	return commitqueue.ClearAllCommitQueues()
 }
 
+func (pc *DBCommitQueueConnector) IsAuthorized(ctx context.Context, username, owner, repo string) (bool, error) {
```

Comment:
I think we should replace three string args with an arg struct to make it more clear.

Reply:
done.